src/Microsoft.SqlTools.ManagedBatchParser/BatchParser/ExecutionEngineCode/BatchParserSqlCmd.cs

```diff
@@ -155,11 +155,7 @@ private string GetFilePath(string fileName)
                     }
 
                 }
-                if (fullFileName == null)
-                {
-                    fullFileName = Path.GetFullPath(fileName);
-                }
-
+                fullFileName ??= Path.GetFullPath(fileName);
                 return fullFileName;
```

Comment:
Similar to the other one:

  return fullFileName ?? Path.GetFullPath(fileName);